applications/daos.md

```diff
@@ -0,0 +1,135 @@
+# W3F Grant Proposal
+***
+
+* **Project Name:** daos
+* **Team Name:** daos-org
+* **Payment Address:** 0x08A7876db92b07057E12fAF29AA3CCF33b5aAe3D (USDT ERC20)
+* **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Overview
+* This project provides a more flexible governance framework for a large number of small groups with common characteristics. It allows developers to create behavior templates based on this set of governance solutions in the simplest way possible,
```

Comment:
What do you mean by behavior and behavior template? Is this what describes the structure of the governance? Are these templates implemented in code, or some kind of markup?

Reply:
> What do you mean by behavior and behavior template? Is this what describes the structure of the governance? Are these templates implemented in code, or some kind of markup?

Behavior templates refer to other templates developed by other developers based on their own business, but this template governance is completely based on daos.

Reply:
> > What do you mean by behavior and behavior template? Is this what describes the structure of the governance? Are these templates implemented in code, or some kind of markup?
> 
> Behavior templates refer to other templates developed by other developers based on their own business, but this template governance is completely based on daos.



> > What do you mean by behavior and behavior template? Is this what describes the structure of the governance? Are these templates implemented in code, or some kind of markup?
> 
> Behavior templates refer to other templates developed by other developers based on their own business, but this template governance is completely based on daos.

For example, if you want to create a room template for a social group, there are many related operations that need to be jointly managed by members of the dao. Then only need to develop according to the document, the developer can quickly create a room dao template with governance, without considering the governance layer. For the user, he only needs to create any number of room dao based on this room template, the origin of each transaction of each dao is completely defined by these dao members themselves

Reply:
> > > What do you mean by behavior and behavior template? Is this what describes the structure of the governance? Are these templates implemented in code, or some kind of markup?
> > 
> > 
> > Behavior templates refer to other templates developed by other developers based on their own business, but this template governance is completely based on daos.
> 
> > > What do you mean by behavior and behavior template? Is this what describes the structure of the governance? Are these templates implemented in code, or some kind of markup?
> > 
> > 
> > Behavior templates refer to other templates developed by other developers based on their own business, but this template governance is completely based on daos.
> 
> For example, if you want to create a room template for a social group, there are many related operations that need to be jointly managed by members of the dao. Then only need to develop according to the document, the developer can quickly create a room dao template with governance, without considering the governance layer. For the user, he only needs to create any number of room dao based on this room template, the origin of each transaction of each dao is completely defined by these dao members themselves

This is a case of using daos to implement [vc dao template](https://github.com/DICO-TEAM/dico-chain/blob/main/runtime/kico/src/vc.rs#L275), users can create vc dao on any asset, including NFT.

Reply:
> What do you mean by behavior and behavior template? Is this what describes the structure of the governance? Are these templates implemented in code, or some kind of markup?


We don't care about the specific implementation of the dao template by developers, we regard these behaviors as some behaviors waiting to be governed. We provide governance solutions and the ability to quickly create dao

Reply:
> the developer can quickly create a room dao template with governance

Just to make sure we are on the same page: to me, creating a room DAO and creating a room DAO _template_ are two quite different things. A template is something that is reused, possibly by others. But for the template to be reusable, it needs to be in a format that others can understand. Looking at the code you linked to, these templates are on-chain data structures that they just need to reference in their DAO creation transaction?

Reply:
> 
Yes, creating a DAO template is different from creating a DAO. DAO templates are created for template developers and DAOs are created for all users who use DAOs

Reply:
> > the developer can quickly create a room dao template with governance
> 
> Just to make sure we are on the same page: to me, creating a room DAO and creating a room DAO _template_ are two quite different things. A template is something that is reused, possibly by others. But for the template to be reusable, it needs to be in a format that others can understand. Looking at the code you linked to, these templates are on-chain data structures that they just need to reference in their DAO creation transaction?

As long as developers follow our requirements when creating the template, a DAO template can be created quickly and users can create their DAOs on the chain based on this template

Reply:
> > the developer can quickly create a room dao template with governance
> 
> Just to make sure we are on the same page: to me, creating a room DAO and creating a room DAO _template_ are two quite different things. A template is something that is reused, possibly by others. But for the template to be reusable, it needs to be in a format that others can understand. Looking at the code you linked to, these templates are on-chain data structures that they just need to reference in their DAO creation transaction?

With DAOS, developers can focus more on the development of template behavior rather than the governance of specific behavior